modules/orchestrator/services/addon/addon_handler.go

```diff
@@ -53,7 +53,7 @@ func (a *Addon) AttachAndCreate(params *apistructs.AddonHandlerCreateItem) (*api
 
 // GetAddonExtention 获取addon的spec，dice.yml信息
 func (a *Addon) GetAddonExtention(params *apistructs.AddonHandlerCreateItem) (*apistructs.AddonExtension, *diceyml.Object, error) {
-	extentionsList, err := a.bdl.QueryExtensionVersions(apistructs.ExtensionVersionQueryRequest{Name: params.AddonName, All: "true"})
+	extentionsList, err := a.bdl.QueryExtensionVersions(apistructs.ExtensionVersionQueryRequest{Name: params.AddonName, All: true})
```

Comment:
I think you should remove all ext related bundle methods together.

Reply:
There are too many bundles using the refactored interface, the next pr will refactor these bundles, and bundles can also test whether the new interface is compatible with the old bundle